changelogs/unreleased/fix-undeleted-unix-socket-path-and-multiple-socket-closing.md

```diff
@@ -0,0 +1,6 @@
+## core/bugfix
+
+ * Fix error, related to the fact, that if user change listen address,
+   all iproto threads closed same socket multiple times.
+   Fix error, related to the fact, that tarantool did not delete the unix
```

Comment:
In the changelogs we use past simple for happened things, according to a recent agreement with the doc team (which I am not sure is persisted anywhere).